ChartsDemo/Swift/Demos/LineChart1ViewController.swift

```diff
@@ -144,31 +144,31 @@ class LineChart1ViewController: DemoBaseViewController {
     override func optionTapped(_ option: Option) {
         switch option {
         case .toggleFilled:
-            for set in chartView.data!.dataSets as! [LineChartDataSet] {
+            for set in chartView.data as! LineChartData {
```

Comment:
want to know which method enables this as well.
And, it seems weird that a `set` object is a `ChartData` type? what you think

Reply:
See comment below

Reply:
@jjatie `for set in chartView.data as! LineChartData` here I'm confused, as `set` implies a dataset class, while it casts to a chartdata class. is the name good here?

Reply:
oh wait. Seems I asked a wrong question?
it's `chartView.data` being casted to `LineChartData`?